lib/utils/settings.js

```diff
@@ -10,9 +10,12 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 import './boot.js';
 
 import { pathFromUrl } from './resolve-url.js';
-export const useShadow = !(window.ShadyDOM);
+export const useShadow = !(window.ShadyDOM) || !(window.ShadyDOM.inUse);
 export const useNativeCSSProperties = Boolean(!window.ShadyCSS || window.ShadyCSS.nativeCss);
 export const useNativeCustomElements = !(window.customElements.polyfillWrapFlushCallback);
+export const supportsAdoptingStyleSheets = useShadow &&
+    ('adoptedStyleSheets' in Document.prototype) &&
+    ('replaceSync' in CSSStyleSheet.prototype);
```

Comment:
Given that the spec is now somewhat in question re: frozen array vs. mutation APIs, would it make sense to try and make this test as defensive as possible in case they end up changing it and need to deprecate the ability to assign an array?  I don't know exactly what it'd be but maybe something like 
```js
const testSheet = new ConstructableStylesheet();
try { document.adoptedStylesheets = [testSheet]; } catch (e) { }
supportsAdoptingStyleSheets = supportsAdoptingStyleSheets && 
  document.adoptedStylesheets[0] === testSheet;
```
(maybe made safe to allow having set some before)?

If so, might also consider making this change to LitElement's detection.

Reply:
Done.